doc/man3/EVP_DigestInit.pod

```diff
@@ -806,7 +806,7 @@ EVP_MD_CTX_update_fn() and EVP_MD_CTX_set_update_fn() were deprecated
 in OpenSSL 3.0.
 
 The functions EVP_MD_CTX_dup() and EVP_DigestSqueeze() were added in
-OpenSSL 3.2.
+OpenSSL 3.3.0.
```

Comment:
Probably the patch version is not necessary here, i.e. just keep it to 3.3

Reply:
Also EVP_MD_CTX_dup() was added in OpenSSL 3.1 - so this needs to be split.

Reply:
Done. Addressed it in latest commit.